services/checks/checks_test.go

```diff
@@ -117,6 +118,33 @@ func TestCollectChecks(t *testing.T) {
 	})
 }
 
+// A proper unit test could not be written due
+// to problems with the code responsible for locating agents
+// Once it is fixed rewrite this test to actually run `executeChecks`
+// method and test for recorded metrics.
+func TestSTTMetrics(t *testing.T) {
+	t.Run("check for recorded metrics", func(t *testing.T) {
+		s := New(nil, nil, nil)
+		expected := strings.NewReader(`
+			# HELP pmm_managed_checks_alerts_generated_total Counter of alerts generated per service type per check type
+        	# TYPE pmm_managed_checks_alerts_generated_total counter
```

Comment:
indentation seems to be messed up on Github, it looks fine on my editor though

Reply:
I guess you mixed tabs and spaces there. Please fix that

Reply:
done @AlekSi

Reply:
done